tsconfig.json

```diff
@@ -1,9 +1,11 @@
 {
     "compilerOptions": {
-      "target": "es2019",
+      "baseUrl": ".",
+      "esModuleInterop": true,
       "module": "esnext",
       "moduleResolution": "node",
-      "esModuleInterop": true
+      "paths": {"*.js": ["*"]},
+      "target": "es2019",
```

Comment:
I actually really prefer `strict` mode in TSConfig, because it helps catch a lot of things. But that would require its own PR probably.